server.js

```diff
@@ -15,6 +15,61 @@ app.get("/", function (request, response) {
 
 // TODO add your routes and helper functions here
 
-const listener = app.listen(process.env.PORT, function () {
+app.get("/booking", function (req, res) {
+  res.send(bookings);
+});
+
+app.get("/booking/:id", function (req, res) {
+  const bookingId = +req.params.id;
```

Comment:
This is a nice way to convert to an int when you know the incoming value will resemble an int, however when you can't be sure it's nice to have some error checking here (e.g. what if the user went to `/booking/abc`?)